mgmtsystem_action/__manifest__.py

```diff
@@ -22,7 +22,6 @@
         'security/mgmtsystem_action_security.xml',
         'data/action_sequence.xml',
         'views/mgmtsystem_action.xml',
-        'views/mgmtsystem_action_kanban.xml',
```

Comment:
Why are you removing this kanban?

Reply:
The only thing we have in this file is the same view kanban view of mgmtsystem.action that is in the mgmtsystem_action.xml file.

https://github.com/OCA/management-system/blob/e3704f65a2126b08023221b4180c3c26d2dde1b0/mgmtsystem_action/views/mgmtsystem_action_kanban.xml#L3

https://github.com/OCA/management-system/blob/e3704f65a2126b08023221b4180c3c26d2dde1b0/mgmtsystem_action/views/mgmtsystem_action.xml#L122

They are exactly the same. It was one of the pylint errors when I pushed the branch for the first time

Reply:
OK, thanks for the explanation